examples/ui/index.xml.njk

```diff
@@ -0,0 +1,13 @@
+---
+hv_title: "UI"
+permalink: "/ui/index.xml"
+hv_tab_id: "tab-ui"
+hv_button_behavior: "none"
+hv_list_tag: "ui_root"
```

Comment:
another Front matter rule that would be worth following, would be to list first the built-in keys, followed by our custom keys, and apply alphabetical indexing for both lists i.e.
```suggestion
permalink: "/ui/index.xml"
hv_button_behavior: "none"
hv_list_tag: "ui_root"
hv_tab_id: "tab-ui"
hv_title: "UI"
```